psdk_wrapper/src/psdk_wrapper.cpp

```diff
@@ -1072,22 +1080,26 @@ PSDKWrapper::initialize_ros_elements()
           std::bind(&PSDKWrapper::camera_get_laser_ranging_info_cb, this, _1,
                     _2),
           qos_profile_);
-  // TODO(@lidiadltv): Enable these actions once are working properly
-  // camera_download_file_list_action_ =
-  //     std::make_unique<nav2_util::SimpleActionServer<CameraDownloadFileList>>(
-  //           shared_from_this(), "psdk_ros2/camera_download_file_list",
-  //           std::bind(&PSDKWrapper::camera_download_file_list_cb,
-  //           this));
-  // camera_download_file_by_index_action_ =
-  //     std::make_unique<nav2_util::SimpleActionServer<CameraDownloadFileByIndex>>(
-  //           shared_from_this(), "psdk_ros2/camera_download_file_by_index",
-  //           std::bind(&PSDKWrapper::camera_download_file_by_index_cb,
-  //           this));
-  // camera_delete_file_by_index_action_ =
-  //     std::make_unique<nav2_util::SimpleActionServer<CameraDeleteFileByIndex>>(
-  //           shared_from_this(), "psdk_ros2/camera_delete_file_by_index",
-  //           std::bind(&PSDKWrapper::camera_delete_file_by_index_cb,
-  //           this));
+  camera_download_file_list_service_ = create_service<CameraDownloadFileList>(
+      "psdk_ros2/camera_download_file_list",
+      std::bind(&PSDKWrapper::camera_download_file_list_cb, this, _1, _2),
+      qos_profile_);
+  camera_download_file_by_index_service_ =
+      create_service<CameraDownloadFileByIndex>(
+          "psdk_ros2/camera_download_file_by_index",
+          std::bind(&PSDKWrapper::camera_download_file_by_index_cb, this, _1,
+                    _2),
+          qos_profile_);
+  camera_delete_file_by_index_service_ =
+      create_service<CameraDeleteFileByIndex>(
+          "psdk_ros2/camera_delete_file_by_index",
+          std::bind(&PSDKWrapper::camera_delete_file_by_index_cb, this, _1, _2),
+          qos_profile_);
+  camera_select_files_by_time_service_ =
```

Comment:
I will add it, I didnt have it because it wasnt in the scope of the task, and the current implementation manages the memory by removing uploaded files